ccp-openshift-scan/scanning/yumupdates.py

```diff
@@ -0,0 +1,39 @@
+# scan script for yum list update scannr
+
+import lib
+
+
+def yum_updates():
+    """
+    Finds yum updates
+    """
+    command = ["yum", "-q", "check-update"]
+    out, err = lib.run_cmd_out_err(command)
+    return out, err
```

Comment:
since the the output is received from another function present in different file, for readability, I kept additional references as to know, whats being returned from called function and whats being passed from `yum_updates` function. Surely, it can be reduced, but I kept it this way for readability.

Reply:
:+1: